Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM quay.io/keboola/aws-cli
 ARG AWS_SECRET_ACCESS_KEY
 ARG AWS_ACCESS_KEY_ID
-RUN /usr/bin/aws s3 cp s3://keboola-drivers/snowflake/snowflake-odbc-2.16.10.x86_64.deb /code/docker/snowflake-odbc.deb
+RUN /usr/bin/aws s3 cp s3://keboola-drivers/snowflake/snowflake-odbc-2.19.3.x86_64.deb /code/docker/snowflake-odbc.deb
```

Comment:
tady by bylo fajn používat stejnou verzi jako v connection - https://github.com/keboola/connection/blob/master/Dockerfile#L4 a stahovat je přímo z repa snowflaku (v tomhle PR bych to ale neřešil)

Reply:
ok, mozem to upravit, nevedel co z toho je preferovany sposob.

Reply:
driver teraz nemenim, skusal som to a dava mi to `SQL execution internal error`
![image](https://user-images.githubusercontent.com/19371734/103771644-66e30500-5028-11eb-87ae-20f927c16a66.png)

Nemam teraz velmi cas zistovat, preco novsia verzia nejde.

Reply:
jj občas jsou ty verze zabugovaný :D proto jsem se snažil používat tu verzi kterou má connection

Reply:
@webrouse ten incident to chce propingnout skrz support - nevadí, že nemáš čas, je to stejně na dlouho - nebude tě to zdržovat hned... ale delší dobu :) https://keboola.atlassian.net/wiki/spaces/SUP/pages/656080918/Snowflake+Incidents

Reply:
ok. https://keboola.atlassian.net/browse/COM-600

Reply:
BTW: asi to nebude verziou ovladaca, uz mi to zacalo padat, aj bez zmeny ovladaca, zatial som to revertol na 3.5.0.